# rc_mqtt_bridge/config/mqtt_bridge.yaml

mqtt_bridge_node:
  ros__parameters:
    # 차량 ID
    car_id: car_01
    
    # MQTT 브로커 설정
    mqtt_host: localhost
    mqtt_port: 1883
    mqtt_username: ""
    mqtt_password: ""
    mqtt_keepalive: 60
    
    # ROS2 토픽
    task_cmd_topic: /server/task_cmd
    task_status_topic: /server/task_status
    emergency_cmd_topic: /control/drive_cmd_emergency
    pose_topic: /localization/pose
    driving_state_topic: /driving/state
    
    # 전송 주기
    location_publish_rate_hz: 5.0   # 위치: 5Hz
    status_publish_rate_hz: 1.0     # 상태: 1Hz
